itou/approvals/admin.py

```diff
@@ -158,6 +157,14 @@ class ApprovalAdmin(admin.ModelAdmin):
         PkSupportRemarkInline,
     )
 
+    def get_form(self, request, obj=None, change=False, **kwargs):
```

Comment:
https://docs.djangoproject.com/en/4.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.get_form
```suggestion
    def get_form(self, request, obj=None, **kwargs):
```